cpp/src/aztec/honk/composer/composer_helper/permutation_helper.hpp

```diff
@@ -1,118 +1,89 @@
-#include <stdint.h>
-#include <stddef.h>
-#include <vector>
+#pragma once
+
+#include <ecc/curves/bn254/fr.hpp>
+#include <polynomials/polynomial.hpp>
 #include <proof_system/proving_key/proving_key.hpp>
+
+#include <algorithm>
+#include <cstdint>
+#include <initializer_list>
+#include <cstdint>
+#include <cstddef>
+#include <utility>
+#include <vector>
+
 namespace honk {
-// Enum values spaced in increments of 30-bits (multiples of 2 ** 30).
-enum WireType { LEFT = 0U, RIGHT = (1U << 30U), OUTPUT = (1U << 31U), FOURTH = 0xc0000000 };
```

Comment:
This was not used directly, and made things a lot more confusing. Instead, we use a `uint32_t` to describe which column we are using explicitly.